src/main/java/entity/Post.java

```diff
@@ -0,0 +1,43 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class Post extends ForumEntry {
+    private String postTitle;
+    private List<Comment> comments;
+    private String category;
```

Comment:
(just commenting this for future reference) maybe we use enum for category instead (if we decide against letting users create their own, predfined categories)

Reply:
That's a good idea, we'll add that in in the future